prompt2model/param_selector/search_with_optuna.py

```diff
@@ -173,6 +173,15 @@ def _build_hp_space(
         if hyperparameter_space is None:
             return DEFAULT_HYPERPARAMETERS_SPACE
         hp_space = {}
+
+        default_keys = list(DEFAULT_HYPERPARAMETERS_SPACE.keys())
+        for key in list(hyperparameter_space.keys()):
+            if key not in default_keys:
+                print(
+                    f"Key {key} is not present in DEFAULT_HYPERPARAMETERS_SPACE. Hence will be ignored",  # noqa
+                    "However, you can expose the key to the Trainer by adding it to DEFAULT_HYPERPARAMETERS_SPACE.",  # noqa
```

Comment:
Don't use bare `noqa` calls, please use `noqa E501` if you're suppressing a "line too long" warning. However, the first line here doesn't actually seem too long (I see that it's 85 characters which is below the 88 character threshold)

Suggestion:
```
                    f"Key {key} is not present in DEFAULT_HYPERPARAMETERS_SPACE. Hence will be ignored",
                    "However, you can expose the key to the Trainer by adding it to DEFAULT_HYPERPARAMETERS_SPACE.",  # noqa E501
```